.ci/ciao_down_setup.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This setup will be used to run the CI tests under ciao-down
+# under SemaphoreCI. This will avoid issues with the old packages
+# that comes in the SempahoreCI environment (Ubuntu 14.04). 
+
+set -e
+
+cidir=$(dirname "$(readlink -f "$0")")
+
+if [ -z $CI ] || [ -z "$LOCALCI_REPO_SLUG" ] || [ -z "$LOCALCI" ]; then
+	echo "	If you are running locally, please set environment variables
+	CI=true
+	LOCALCI=true
+	LOCALCI_REPO_SLUG with the value of the repo you want to test.
+	For example:
+	export LOCALCI_REPO_SLUG=clearcontainers/tests
+	If you want to test an specific PR, please also add env variable:
+	LOCALCI_PR_NUMBER, for example:
+	export LOCALCI_PR_NUMBER=100" && exit 1
+fi
+
+ciao_repo="github.com/01org/ciao"
+
+# This function will clone the repo inside the VM and
+# get the current PR that will be tested.
+function install_repo_inside_vm(){
+	export $(cat "${HOME}/ci_environment")
+	repo_slug="$LOCALCI_REPO_SLUG"
+	pr_number="$LOCALCI_PR_NUMBER"
+	if [ -n "$SEMAPHORE" ]; then
+		repo_slug="$SEMAPHORE_REPO_SLUG"
+		pr_number="$PULL_REQUEST_NUMBER"
+	fi
+	go get -d "github.com/${repo_slug}" || true
+	if [ -n "$pr_number" ]; then
+		pushd "${GOPATH}/src/github.com/${repo_slug}"
+		git fetch origin "refs/pull/${pr_number}/head:pull_${pr_number}"
+		git checkout "pull_${pr_number}"
+	fi
+}
+
+# Install ciao-down dependencies
+source /etc/os-release
```

Comment:
Taking a look at the documentation, I just saw that `ciao-down` installs the dependencies for us. It supports Fedora, Ubuntu and Clear. 
https://github.com/01org/ciao/blob/master/testutil/ciao-down/deps.go 
I tried on a fresh Ubuntu and worked well. 
So I'll just remove these lines, is that ok ? 

```
cloud@test-deleteme:~/go/src/github.com/01org/ciao$ $GOPATH/bin/ciao-down create xenial
Installing host dependencies
OS Detected: ubuntu
Missing packages detected: [qemu-system-x86 qemu-utils xorriso]
Reading package lists...
Building dependency tree...
Reading state information...
The following package was automatically installed and is no longer required:
  snap-confine
Use 'sudo apt autoremove' to remove it.
The following additional packages will be installed:
  cpu-checker ipxe-qemu libaio1 libasound2 libasound2-data libasyncns0
  libbluetooth3 libboost-iostreams1.58.0 libboost-random1.58.0
  libboost-system1.58.0 libboost-thread1.58.0 libbrlapi0.6 libburn4 libcaca0
  libcacard0 libfdt1 libflac8 libiscsi2 libisoburn1 libisofs6 libjpeg-turbo8
  libjpeg8 libjte1 libnspr4 libnss3 libnss3-nssdb libogg0 libopus0
  libpixman-1-0 libpulse0 librados2 librbd1 libsdl1.2debian libsndfile1
  libspice-server1 libusbredirparser1 libvorbis0a libvorbisenc2 libxen-4.6
  libxenstore3.0 libyajl2 msr-tools qemu-block-extra qemu-system-common
  seabios sharutils
```